borg/archiver.py

```diff
@@ -1018,6 +1025,9 @@ def build_parser(self, args=None, prog=None):
                                type=ChunkerParams, default=CHUNKER_PARAMS,
                                metavar='CHUNK_MIN_EXP,CHUNK_MAX_EXP,HASH_MASK_BITS,HASH_WINDOW_SIZE',
                                help='specify the chunker parameters. default: %d,%d,%d,%d' % CHUNKER_PARAMS)
+        subparser.add_argument('--ignore-inode', dest='ignore_inode',
+                               action='store_true', default=False,
```

Comment:
I think `action='store_true', default=False,` is redundant, but it's always used here. Maybe something for a little cleanup session.